views/jobs.ejs

```diff
@@ -0,0 +1,37 @@
+<h2>Jobs List</h2>
+<a href="/jobs/new">
+  <button type="button">Add New Job</button>
+</a>
+<table id="jobs-table">
+  <tr id="jobs-table-header">
+    <th>Company</th>
+    <th>Position</th>
+    <th>Status</th>
+    <th colspan="2">Actions</th>
+  </tr>
+  <% if (jobs && jobs.length) { %>
+    <% jobs.forEach((job) => { %>
+      <tr>
+        <td><%= job.company %></td>
+        <td><%= job.position %></td>
+        <td><%= job.status %></td>
+        <td>
+            <a href="/jobs/edit/<%= job.id %>">
+              <button type="button">edit</button>
```

Comment:
Requires change:
It's anti-pattern to a clickable element wrapping another clickable element. What do you want the action to be? A link that sends you to `/jobs/edit/:id` or a button that edits the current job? In this case, remove the button element and instead make the **link** tag LOOK like a button using CSS!